web_src/less/_repository.less

```diff
@@ -998,6 +998,43 @@
             border-bottom-left-radius: 4px;
             border-bottom-right-radius: 4px;
           }
+
+          > .merge-section {
```

Comment:
Can we leave the code as it was, and remove the `>`? I don't think the `>` really helps .... Maybe one day it causes a new bug.